src/main/webapp/app/admin/cleanup-service/cleanup-service.component.ts

```diff
@@ -0,0 +1,108 @@
+import { Component, OnInit } from '@angular/core';
+import dayjs from 'dayjs/esm';
+import { CleanupOperation } from 'app/admin/cleanup-service/cleanup-operation.model';
+import { CleanupServiceExecutionRecordDTO, DataCleanupService } from 'app/admin/cleanup-service/cleanup-service.service';
+import { convertDateFromClient, convertDateFromServer } from 'app/utils/date.utils';
+import { Subject } from 'rxjs';
+import { HttpErrorResponse, HttpResponse } from '@angular/common/http';
+
+@Component({
+    selector: 'jhi-cleanup-service',
+    templateUrl: './cleanup-service.component.html',
+})
+export class CleanupServiceComponent implements OnInit {
+    private dialogErrorSource = new Subject<string>();
+    dialogError = this.dialogErrorSource.asObservable();
+
+    constructor(private cleanupService: DataCleanupService) {}
```

Comment:
Could we use the `inject` function instead?

```suggestion
    private cleanupService = inject(DataCleanupService);
```

Reply:
done